### PR TITLE
Fix man page for command not optional  (#87)

### PR DESCRIPTION
**Pull request by swadeley**
Hello

Please review two changes to yum man page.

Thank you

### CODE REVIEW

**Review by Conan-Kudo — APPROVED** [review]

File: docs/yum.8 [hunk 1]
```diff
@@ -858,12 +858,12 @@ the installroot (and thus. will be empty, before creation).
 .br
 Configuration Option: \fBinstallroot\fP
 .IP "\fB\-\-enablerepo=repoidglob\fP"
-Enables specific repositories by id or glob that have been disabled in the 
+Temporarily enables specific repositories by id or glob that have been disabled in the
```
Comment (dmnks) [inline docs/yum.8:861]: Based on my comment in the related BZ https://bugzilla.redhat.com/show_bug.cgi?id=1573154#c5, I propose that we omit this hunk and just use the very first one (`[command]` -> `COMMAND`).

File: docs/yum.8 [hunk 2]
```diff
@@ -858,12 +858,12 @@ the installroot (and thus. will be empty, before creation).
...
 Configuration Option: \fBinstallroot\fP
 .IP "\fB\-\-enablerepo=repoidglob\fP"
-Enables specific repositories by id or glob that have been disabled in the 
+Temporarily enables specific repositories by id or glob that have been disabled in the 
 configuration file using the enabled=0 option.
 .br
 Configuration Option: \fBenabled\fP
 .IP "\fB\-\-disablerepo=repoidglob\fP"
-Disables specific repositories by id or glob. 
+Temporarily disables specific repositories by id or glob.
```
Comment (dmnks) [inline docs/yum.8:866]: Same as above.

**Review by dmnks — CHANGES_REQUESTED** [review]

File: docs/yum.8 (hunk 1 above)
Comment (swadeley) [reply docs/yum.8:861]: OK, I presume I do not need to push a change, you can just not use it.

File: docs/yum.8 (hunk 2 above)
Comment (swadeley) [reply docs/yum.8:866]: OK, I presume I do not need to push a change, you can just not use it.

File: docs/yum.8 (hunk 1 above)
Comment (dmnks) [reply docs/yum.8:861]: Yeah, I just noticed they are separate commits. Thanks, I'll do the honors then.

File: docs/yum.8 (hunk 2 above)
Comment (dmnks) [reply docs/yum.8]: Yeah, I just noticed they are different commits. I'll do the honors then, thanks.

**Comment by dmnks** [comment]
I have merged the first commit (the `[command]` -> `COMMAND` change) but I did that manually without referencing this PR, so I'm closing it manually. Thanks for the contribution!

**Comment by swadeley** [comment]
ACK, thanks to all.

**Pull request closed without merging**